lib/Conversion/TritonGPUToLLVM/TensorMemRefOpToLLVM.cpp

```diff
@@ -0,0 +1,120 @@
+#include "mlir/IR/Matchers.h"
+#include "mlir/IR/TypeUtilities.h"
+#include "mlir/Conversion/LLVMCommon/MemRefBuilder.h"
+
+#include "TensorMemRefOpToLLVM.h"
+
+using namespace mlir;
+using namespace mlir::triton;
+
+using ::mlir::LLVM::getElementsFromStruct;
```

Comment:
One more thing: `getElementsFromStruct` and `getStructFromElements` are deprecated in [openai#1281](https://github.com/ROCmSoftwarePlatform/triton/commit/0f82fac60e3b6833e96046a91687dd91f25c489b). Could you rebase your branch on the latest `dot-rocMLIR` and apply the new `[un]packLLElements`?

Reply:
Will change them.